prow/config.yaml

```diff
@@ -132,6 +132,10 @@ branch-protection:
           restrictions: # only allow admins
             users: []
             teams: []
+        k8s.io:
+          required_status_checks:
+            contexts:
+            - pull-k8sio-cip
```

Comment:
You shouldn't ever need to explicitly require a ProwJob context. That is automatically inferred from the ProwJob config. The `pull-k8sio-cip` job is not already required because it is a `run_if_changed` job and doesn't necessarily run so requiring it would prevent some PRs from merging because the job would never run.
Note that Tide is aware of the job and won't merge if the PR should run the job based on the regexp, but is missing the context entirely for some reason.

Today we post a "Skipped." status if the job doesn't need to run on a PR, but that is legacy behavior and will be disabled soon.

Reply:
Hmm so this whole chunk is unnecessary?

I'm testing this live at https://github.com/kubernetes/k8s.io/pull/212 --- the pull-k8sio-cip job failed there. Are you saying that /approve won't do anything because the check failed?

Reply:
Correct. It looks like the PR needs 
> /lgtm
> /approve
> /hold cancel

But after that the `tide` status context will say that `pull-k8sio-cip` needs to be passing and Tide won't merge the PR until it is.

Reply:
Gotcha --- will update this PR.

Also, I am still new to Prow and am going to assume that by `tide` status context, you mean the "Details" link next to the `tide` check.

Reply:
I'm referring to the description next to that status context (currently `Not mergeable. Needs approved, lgtm labels.`), but the "Details" link will take you to the PR status page which should show the same information.